admin/class-paystack-forms-admin.php

```diff
@@ -382,6 +382,9 @@ function kkd_pff_paystack_editor_add_form_data()
 						<option value="GHS" ' . kkd_pff_paystack_txncheck('GHS', $currency) . '>Ghanaian Cedis</option>
 						<option value="ZAR" ' . kkd_pff_paystack_txncheck('ZAR', $currency) . '>South African Rand</option>
 						<option value="KES" ' . kkd_pff_paystack_txncheck('KES', $currency) . '>Kenyan Shillings</option>
+                        <option value="XOF" ' . kkd_pff_paystack_txncheck('XOF', $currency) . '>West African CFA franc</option>
+                        <option value="RWF" ' . kkd_pff_paystack_txncheck('RWF', $currency) . '>Rwandan Francs</option>
```

Comment:
Should be Rwandan *Franc to be consistent in listing currencies in the singular form

Reply:
Also update CFA fran to CFA *Franc (capitalize the F)